### PR TITLE
#133: [Dropdown] export react-select style from /dropdown (closes #133)

### LINKED ISSUE #133: [Dropdown] export react-select style from /dropdown
## requirements
- facilitate import of needed style
- make it more clear that this component comes with a `css` file
- follow our (soon to be) sass name convention: `camelCase` version of the component name (refer to point in #204 )

## specs
manually copy `node_modules/react-select/dist/default.css` to `dropdown/dropdown.scss`

[EDITED]

### PR DESCRIPTION
**Pull request by nerder**
Issue #133

### CODE REVIEW

**Comment by nerder** [comment]
NOTE: please before to merge this, make sure to close #324

File: src/dropdown/Dropdown.js [hunk 1]
```diff
@@ -5,6 +5,8 @@ import cx from 'classnames';
 import { props, t } from '../utils';
 import { warn } from '../utils/log';
 
+import './dropdown.scss';
```
Comment (FrancescoCioria) [inline src/dropdown/Dropdown.js]: This is a breaking change we don't want: every old project built without webpack is going to break if you add this import.
Please remove

File: src/dropdown/dropdown.scss [hunk 2]
```diff
@@ -0,0 +1,276 @@
+/**
```
Comment (FrancescoCioria) [inline src/dropdown/dropdown.scss]: this file should be named `style.scss`

File: src/dropdown/Dropdown.js (hunk 1 above)
Comment (nerder) [inline src/dropdown/Dropdown.js]: i saw this in other components, how am i suppose to import `style` then?

Comment (FrancescoCioria) [inline src/dropdown/Dropdown.js]: you don't, the user will import it by himself (like we're currently doing)
ex:
```js
import 'buildo-react-components/src/dropdown/style.scss'
```

File: src/dropdown/dropdown.scss (hunk 2 above)
Comment (FrancescoCioria) [inline src/dropdown/dropdown.scss]: wait

Comment (FrancescoCioria) [inline src/dropdown/dropdown.scss]: I updated the specs: leave as is.

**Comment by FrancescoCioria** [comment]
@nerder why does it have so many commits now? this morning it had only the last two (the correct ones)

**Comment by nerder** [comment]
i mess up with rebase i guess, need to cleanup

**Comment by nerder** [comment]
Can be merged now @FrancescoCioria

File: src/dropdown/Dropdown.js [hunk 3]
```diff
@@ -5,6 +5,8 @@ import find from 'lodash/find';
 import cx from 'classnames';
 import { warn } from '../utils/log';
 
+import './dropdown.scss';
```
Comment (FrancescoCioria) [inline src/dropdown/Dropdown.js]: why is this still here? please remove

File: examples/components/styles.js [hunk 4]
```diff
@@ -1,6 +1,7 @@
 import '../../src/more-or-less/moreOrLess.scss';
 import '../../src/flex/flexView.scss';
 import '../../src/loading-spinner/style.scss';
+import '../../src/dropdown/dropdown.scss';
 import '../../src/confirmation-input/confirmationInput.scss';
 import '../../src/badge/badge.scss';
 import '../../src/toggle/toggle.scss';
```
Comment (FrancescoCioria) [inline examples/components/styles.js]: you should remove import of the old style: `import '../../node_modules/react-select/dist/default.css';`
(it's two lines below but I can't comment on it)

Comment (nerder) [inline examples/components/styles.js]: Completely didn't seen. Thank you for that.

PS: you want me to add here the css mentioned in #108 on in another pr?

Comment (FrancescoCioria) [inline examples/components/styles.js]: better in another PR and it should be added to the main style (`dropdown.scss`)

Comment (nerder) [inline examples/components/styles.js]: ok i'll open another issue then

**Comment by FrancescoCioria** [comment]
:vhs:

**Comment by nerder** [comment]
:hand:

**Pull request merged**